prediction_market_agent/agents/known_outcome_agent/deploy.py

```diff
@@ -80,8 +82,18 @@ def answer_binary_market(self, market: AgentMarket) -> bool | None:
         return None
 
     def calculate_bet_amount(self, answer: bool, market: AgentMarket) -> BetAmount:
-        if market.currency == Currency.xDai:
-            return BetAmount(amount=Decimal(0.1), currency=Currency.xDai)
+        if isinstance(market, OmenAgentMarket):
+            if market.currency != Currency.xDai:
+                should_not_happen()
+            return BetAmount(
+                # On markets without liquidity, bet just a small amount for benchmarking.
+                amount=(
+                    Decimal(0.1)
```

Comment:
With the liquidity check in place, maybe this could be higher?

Reply:
Yes, bump to 1.0 xDai!!

Reply:
Bumped 💪 💪